scipy/stats/_stats_py.py

```diff
@@ -9710,14 +9719,62 @@ def wasserstein_distance(u_values, v_values, u_weights=None, v_weights=None):
     :math:`\mathbb{R} \times \mathbb{R}` whose marginals are :math:`u` and
     :math:`v` on the first and second factors respectively.
 
-    If :math:`U` and :math:`V` are the respective CDFs of :math:`u` and
-    :math:`v`, this distance also equals to:
+    In the 1-dimensional case, let :math:`U` and :math:`V` denote the
+    respective CDFs of :math:`u` and :math:`v`, this distance also equals to:
 
     .. math::
 
         l_1(u, v) = \int_{-\infty}^{+\infty} |U-V|
 
-    See [2]_ for a proof of the equivalence of both definitions.
+    See [3]_ for a proof of the equivalence of both definitions.
+
+    In the more general (higher dimensional) and discrete case, it is also
+    called the optimal transport problem or the Monge problem, which can be
+    expressed in this form,
+
+    Given :math:`\Gamma` the transport plan, :math:`D` the distance matrix and
+    follows,
+
+    .. math::
+
+        x = vec(\Gamma)
+        c = vec(D)
+        b = \begin{bmatrix}
+                u\\
+                v\\
+            \end{bmatrix}
```

Comment:
This renders like:
![image](https://user-images.githubusercontent.com/6570539/224386598-1c47ab35-2256-40c8-a40f-745540139d9f.png)

Reply:
Thanks I will add line breaks

Reply:
That helped, but what does $vec$ mean? Currently, it renders as though it were the product of variables $v$, $e$, and $c$. If it is a function, it should not be italicized (e.g. $\sin x, \log x$), and I'd suggest defining what $\mbox{vec} \left( \cdot \right)$ means.